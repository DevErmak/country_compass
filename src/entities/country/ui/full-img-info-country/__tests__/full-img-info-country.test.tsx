import { renderComponentWithStore } from '@/shared/lib/test-helpers';
import FullImgInfoCountry from '..';

it('renders correctly home', () => {
  const { container } = renderComponentWithStore({
    Component: FullImgInfoCountry,
    preloadedState: {
      infoCountries: {
        listCountries: [],
        fullInfoCountry: [],
        isFullInfoCountry: false,
        isLoading: false,
        infoErrorResponse: '',
      },
      infoUser: {
        userName: '',
        listFavoriteCountries: [],
        isAuthentication: false,
        infoModal: { isActiveModal: false, formModal: 'login' },
      },
    },
  });
  expect(container).toMatchSnapshot();
});
