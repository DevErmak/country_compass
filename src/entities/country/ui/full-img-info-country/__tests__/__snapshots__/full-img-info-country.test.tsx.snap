// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly home 1`] = `
<div>
  <div
    class="container-img-full-info-country"
  >
    <div
      class="flag"
    >
      Flag
    </div>
    <img
      class="flag-img"
    />
    <div
      class="coat-of-arms"
    >
      Coat of arms
    </div>
    <img
      alt=""
      class="coat-of-arms-img"
    />
  </div>
  <div
    id="portal"
  />
</div>
`;
